src/Microsoft.ML.Tokenizers/Model/EnglishRoberta.cs

```diff
@@ -27,6 +27,7 @@ public sealed class EnglishRoberta : Model
         private readonly IReadOnlyDictionary<char, char> _unicodeToByte;
         private readonly string[] _charToString;
         private readonly Cache<string, IReadOnlyList<Token>> _cache;
+        private readonly char _startChar = '\u0120';
```

Comment:
```suggestion
        private const char StartChar = (char)(' ' + 256);
```
I found this a helpful explanation of why this is so: https://github.com/openai/gpt-2/issues/80#issuecomment-487202159

Seems like we have a similar function here which does the 256 shift: https://github.com/dotnet/machinelearning/blob/51d8dc47480743b8c30b8622e3c3f094d76c44ef/src/Microsoft.ML.Tokenizers/Model/EnglishRoberta.cs#L458-L482

Might be good to explain how these are connected.

Reply:
We do have a function that does a 256 shift, but this is specifically shifting the "start" token so we can use it for comparisons and stuff. I'll add a note about it though.